.github/workflows/release-typescript-models.yaml

```diff
@@ -0,0 +1,87 @@
+ 
+# Release a typescript package to npm containing the typescript types generated from the latest merged crds
+name: types
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  release-typescript-models:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout devfile/api
+        uses: actions/checkout@v2
+        with:
+          path: api
+
+      - name: Checkout kubernetes-client/gen
+        uses: actions/checkout@v2
+        with:
+          repository: kubernetes-client/gen
+          path: gen
+          ref: 5c6d90b260fd94af32157f304f971778c899b5e2
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9.2'
+
+      - name: Install Python dependencies
+        uses: py-actions/py-dependency-install@v2
+        with:
+          path: "api/.github/actions/generate_types/requirements.txt"
+
+      - name: Generate openapi-generator compatible swagger.json 
+        run: |
+          python .github/actions/generate_types/generate.py
+          mkdir -p /tmp/typescript-models
+          mv swagger.json /tmp/typescript-models/swagger.json.unprocessed
+        working-directory: api
+
+      - name: Create empty client-gen configuration
+        run: |
+            {
+              echo 'export KUBERNETES_BRANCH=""' 
+              echo 'export CLIENT_VERSION=""' 
+              echo 'export PACKAGE_NAME=""'
+              echo 'export USERNAME=""'
+              echo 'export REPOSITORY=""'
+            } >> config.sh
+        working-directory: /tmp
+
+      - name: Generate the typescript models
+        run: |
+          # Remove the contents of custom objects spec so that we aren't bundling any extra objects
+          echo "{}" > custom_objects_spec.json
+          export OPENAPI_SKIP_FETCH_SPEC=true
+          ./typescript.sh /tmp/typescript-models /tmp/config.sh
+        working-directory: gen/openapi
+
+      - name: Modify package.json
+        run: |
+          sed -i 's/\"name\": \".*\"/"name": "@devfile\/typescript-types"/g' /tmp/typescript-models/package.json
```

Comment:
could it be just @devfile/api ?

Reply:
Does it makes sense to separate types from client?
If no: eventually we'll create a common devworkspace typescript client, so, we may want to go with `devfile/node-client`. I was inspiring with https://kubernetes.io/docs/reference/using-api/client-libraries/ =)

Reply:
AFAIS it's changed to devfile/api